exercise/static/exercise/results_staff.js

```diff
@@ -1174,8 +1174,12 @@
              */
             function renderParticipantLink(data, type, row) {
                 // TODO: Get the link to students in a proper way
-                const link = $('li.menu-participants').find('a').attr('href');
-                return (row['UserID'] > 0 ? '<a href="' + link + row['UserID'] + '">' + data + '</a>' : '');
+                if(data == '—') {
```

Comment:
A comment about the coding style:

This older JS file does not follow our coding style guide.
https://apluslms.github.io/contribute/styleguides/

This file can keep using the old style, therefore, the indentation with 4 spaces is OK even though the new style was defined with 2 spaces. Using consistent style inside a file is smart.

P.S. I have never liked `if` keywords with no space after them... `if (somecondition) { ...`